packages/patrol/lib/src/common.dart

```diff
@@ -237,19 +238,34 @@ String deduplicateGroupEntryName(String parentName, String currentName) {
   );
 }
 
-/// Recursively prints the structure of the test suite.
+/// Recursively prints the structure of the test suite and reports test count
+/// of the top-most group
 @internal
-void printGroupStructure(DartGroupEntry group, {int indentation = 0}) {
+int reportGroupStructure(DartGroupEntry group, {int indentation = 0}) {
```

Comment:
This function now also calls `postEvent() from `dart:developer`, update the comment accordingly.

Reply:
the comment says `reports test count of the top-most group`, isn't this enough?